script/contextmenu.js

```diff
@@ -1,4 +1,20 @@
 /**
+ * create an iterable array of selected cells from the selection object
+ *
+ * @param selection object
+ * @returns {Array}
+ */
+var cellArray = function (selection) {
```

Comment:
Since this is declared in the global namespace it should contain our plugin name to avoid possible name clashes with other scripts. ```editTableCellArray``` maybe?